fastplotlib/plot.py

```diff
@@ -103,4 +104,57 @@ def show(self, autoscale: bool = True):
         if autoscale:
             self.auto_scale(maintain_aspect=True, zoom=0.95)
 
-        return self.canvas
+        if toolbar:
+            tools = ToolBar(self).toolbar
+            return VBox([self.canvas, tools])
+        else:
+            return self.canvas
+
+
+class ToolBar:
+    def __init__(self,
+                 plot: Plot):
+        """
+        Basic toolbar for a Plot instance.
+
+        Parameters
+        ----------
+        plot: encapsulated plot instance that will be manipulated using the toolbar buttons
+        """
+        self.plot = plot
+
+        self.autoscale_button = Button(value=False, disabled=False, icon='expand-arrows-alt',
+                                       layout=Layout(width='auto'), tooltip='Auto-scale the camera w.r.t to the scene')
```

Comment:
simpler tool tips, just "auto-scale scene" is enough, people don't need to be thinking about cameras